pkg/reconciler/autoscaling/hpa/resources/keda.go

```diff
@@ -128,6 +131,20 @@ func DesiredScaledObject(ctx context.Context, pa *autoscalingv1alpha1.PodAutosca
 				return nil, fmt.Errorf("query is missing for custom metric: %w", err)
 			}
 
+			tmpl, err := template.New("query").Funcs(sprig.TxtFuncMap()).Parse(query)
```

Comment:
Sprig has more functions but for now let's keep this minimal, just use the Golang templates.

Reply:
I removed `sprig`.